.github/workflows/dotnet-deploy.yml

```diff
@@ -40,7 +40,8 @@ jobs:
       - name: Set up .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: ${{ env.DOTNET_VERSION }}
+          dotnet-version: 6.0.x
+          include-prerelease: true

```

Comment:
Why use prerelease version? Latest stable should be the way to go imho. Keeping a project on a preview version when everyone is locally building on a stable version may be a potential cause of issues.

Reply:
As far as I know, you still need to set `include-prerelease` to get .NET 6 in a pipeline. I know, sounds strange.

Reply:
Where exactly does it cause problems? I'll use the most typical excuse called "works on my machine" 😅 But seriously, I don't have problems with not using it in my private repos 😕 

After doing a bit research on the internet, people seem to use it without it as well, see [here](https://dev.to/kritner/cicd-for-net-6-using-github-actions-1kln)

Reply:
Understandable, let's ditch it, then.

Reply:
exactly as @petrspelos said, it's an old habit from times when it was required to mention prerelease :/ 
Thank you for noticing that :)